### DISCUSSION TITLE
Abort_on_no_activity

### DISCUSSION THREAD

**Discussion by adasi11** [rsachetto/MonoAlg3D_C#48] [Q&A] [answered]
Hey, 

I was wondering what's the difference between abort_on_no_activity and vm_threshold? 
Are they complementary functions?
Is abort_on_no_activity Vm-dependent? 

I have set abort_on_no_activity=true but it doesn't stop the simulation. 
Should I take into account the more positive resting membrane potential of the atria? Like:

abort_on_no_activity=true
vm_threshold=-70

**Comment by rsachetto** [comment, marked as answer]
They are complementary parameters. When all cells have Vm lower or equal to the threshold, the simulation will stop.
